cmd/podman/varlink/io.podman.varlink

```diff
@@ -809,7 +809,7 @@ method DeleteUnusedImages() -> (images: []string)
 # container while it is being committed, pass a _true_ bool for the pause argument.  If the container cannot
 # be found by the ID or name provided, a (ContainerNotFound)[#ContainerNotFound] error will be returned; otherwise,
 # the resulting image's ID will be returned as a string.
```

Comment:
Code says moreResponse not image ID will be returned.

Reply:
fixed!